Makefile

```diff
@@ -190,26 +189,6 @@ project_graph_report:
 		dot -Tsvg merged-graph-rev-sub.gv -o merged-graph-rev-sub.svg && \
 		echo "See artifacts in build/reports/dependency-analysis"
 
-build_android_image:
-	cd ./ci/docker/android-builder && \
-	docker build -t android-builder .
-
-# new ANDROID_BUILDER_TAG will be printed after successful publishing
-internal_publish_android_builder:
-	docker run --rm \
-        --volume /var/run/docker.sock:/var/run/docker.sock \
-        --volume "$(shell pwd)/ci/docker/android-builder":/build \
-        --env DOCKER_REGISTRY=$(DOCKER_REGISTRY) \
-        --env DOCKER_LOGIN=$(DOCKER_LOGIN) \
-        --env DOCKER_PASSWORD=$(DOCKER_PASSWORD) \
-        ${IMAGE_DOCKER_IN_DOCKER} publish_docker_image publish /build
-
-# run after new ANDROID_BUILDER_TAG set
-# assume `docker login` to dockerhub was successful
-publish_android_builder:
```

Comment:
Removed because these scripts are not used in CI and only duplicate scripts in ci/docker.
I want to have one source of truth.